frontend/src/metabase/nav/containers/MainNavbar/MainNavbarView.tsx

```diff
@@ -85,18 +87,34 @@ function MainNavbarView({
   return (
     <SidebarContentRoot>
       <div>
+        <SidebarSection>
+          <ul>
+            <HomePageLink
+              isSelected={
+                selectedItem.type === "non-entity" && selectedItem.url === "/"
+              }
+              icon="home"
+              onClick={onItemSelect}
+              url="/"
+            >
+              {t`Home page`}
+            </HomePageLink>
+          </ul>
+        </SidebarSection>
+
         {bookmarks.length > 0 && (
           <SidebarSection>
             <BookmarkList
               bookmarks={bookmarks}
               selectedItem={
-                selectedItem.type !== "unknown" ? selectedItem : undefined
+                selectedItem.type === "non-entity" ? selectedItem : undefined
```

Comment:
We actually need selected entity item here

Reply:
Whoa, good catch. Don't know how this one happened. 🙏